src/errors.ts

```diff
@@ -53,16 +53,31 @@ export class TimeoutError extends GenericError {
  */
 export class PopupTimeoutError extends TimeoutError {
   constructor(public popup: Window) {
-    super();
+    super() /* istanbul ignore next */;
     //https://github.com/Microsoft/TypeScript-wiki/blob/master/Breaking-Changes.md#extending-built-ins-like-error-array-and-map-may-no-longer-work
     Object.setPrototypeOf(this, PopupTimeoutError.prototype);
   }
 }
 
 export class PopupCancelledError extends GenericError {
   constructor(public popup: Window) {
-    super('cancelled', 'Popup closed');
+    super('cancelled', 'Popup closed') /* istanbul ignore next */;
     //https://github.com/Microsoft/TypeScript-wiki/blob/master/Breaking-Changes.md#extending-built-ins-like-error-array-and-map-may-no-longer-work
     Object.setPrototypeOf(this, PopupCancelledError.prototype);
   }
 }
+
+/**
+ * Error thrown when the token exchange results in a `mfa_required` error
+ */
+export class MfaRequiredError extends GenericError {
```

Comment:
Have you managed to test using this in a TypeScript app? I think we will need to export this error type here if we want to use it outside the library: https://github.com/auth0/auth0-spa-js/blob/cd3b1f9e9c88ce884251f8d1bb4b2550f70527a6/src/index.ts#L27-L33

Reply:
And here: https://github.com/auth0/auth0-spa-js/blob/cd3b1f9e9c88ce884251f8d1bb4b2550f70527a6/src/index.cjs.ts#L1-L7